sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/rx/WebExceptionRetryPolicyE2ETests.java

```diff
@@ -0,0 +1,303 @@
+package com.azure.cosmos.rx;
+
+
+import com.azure.cosmos.BridgeInternal;
```

Comment:
have you pushed all the changes? in this PR I have only seen the tests, but not the real changes

Reply:
Added them now.